caste.go

```diff
@@ -479,6 +534,12 @@ func ToUintE(i interface{}) (uint, error) {
 		return 0, nil
 	case nil:
 		return 0, nil
+	case fmt.Stringer:
+		v, err := strconv.ParseUint(s.String(), 10, 64)
```

Comment:
Is this relevant to this particular PR?

Reply:
`ParseUint` is relevant
`fmt.Stringer`? no, I changed it on `json.Number`